deepchem/models/torch_models/ferminet.py

```diff
@@ -455,7 +462,7 @@ def prepare_hf_solution(self):
                 self.nucleon_coordinates[i][1][0]) + " " + str(
                     self.nucleon_coordinates[i][1][1]) + " " + str(
                         self.nucleon_coordinates[i][1][2]) + ";"
-        self.mol = pyscf.gto.Mole(atom=molecule, basis='sto-3g')
+        self.mol = pyscf.gto.Mole(atom=molecule, basis='sto-6g')
```

Comment:
This is a minor bump of HF orbital function from sto-3g to sto-6g for the improved baselines. Very minor change, hence adding it in this PR instead of a separate cleanup PR.